### PR TITLE
update to 1.6.0

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender. :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,6 +32,8 @@ requirements:
     - poyo >=0.1.0
     - python
     - whichcraft >=0.4.0
+    - requests >=2.18.0
+    - pytest-mock
```
Comment (bollwyvl) [inline recipe/meta.yaml:36]: Is this really a runtime dependency now?

**Comment by bollwyvl** [comment]
Oooh, this looks kind of insidious.

The py27 fails (which i can reproduce locally) have stuff like:
```
E       fixture 'mocker' not found
>       available fixtures: cache, capfd, caplog, capsys, capturelog, clean_system, cov, disable_poyo_logging, doctest_namespace, monkeypatch, pytestconfig, record_xml_property, recwarn, tmpdir, tmpdir_factory, user_config_data, user_config_file, user_dir
>       use 'pytest --fixtures [testpath]' for help on them.
```

Looking at what actually gets installed... no `mock`. Adding `mock # [py2k]` explicitly makes it work locally. And indeed, looking in the current tarball (https://anaconda.org/conda-forge/pytest-mock/1.6.3/download/noarch/pytest-mock-1.6.3-py_0.tar.bz2) i see in `index.json`:
```json
{
  "arch": null,
  "build": "py_0",
  "build_number": 0,
  "depends": [
    "pytest",
    "python"
  ],
  "license": "MIT",
  "name": "pytest-mock",
  "noarch": "python",
  "platform": null,
  "subdir": "noarch",
  "version": "1.6.3"
}
```

This looks like [these](https://github.com/conda-forge/pytest-mock-feedstock/issues/12) [issues](https://github.com/conda/conda-build/issues/1024), perhaps...

**Comment by jakirkham** [comment]
Nice find @bollwyvl! Could you please raise an issue at the `pytest-mock` feedstock?

**Comment by bollwyvl** [comment]
Looks like this has been resolved!

**Pull request closed without merging**